src/styles/styles.css

```diff
@@ -84,31 +114,12 @@
     margin-right: 0px !important;
     margin-left: 0px !important;
 }
-#leftcontent {
-    font-family: 'Roboto';
-    font-weight: 400;
-    font-size: 14px;
-    color: #485465;
-    letter-spacing: -0.08px;
-    text-align: center;
-}
-#rightcontent {
-    font-family: 'Roboto';
-    font-weight: 400;
-    font-size: 14px;
-    color: #485465;
-    letter-spacing: -0.08px;
-    text-align: center;
-}
-#overall-title {
-    background: #FFFFFF;
-    box-shadow: 0 2px 7px 0 rgba(0,0,0,0.16);
-    height: 60px;
-    margin-bottom: 20px
-}
 #container {
     padding-bottom: 20px;
     background: #FAFAFA !important;
+    box-shadow: 0 2px 7px 0 rgb(0 0 0 / 16%);
```

Comment:
Is this needed?

Reply:
Yes box-shadow is needed for the outer shadow of the container. Previously it was loaded from external stylesheet.